public/css/login.css

```diff
@@ -0,0 +1,59 @@
+* {
+  padding: 0%;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto;
+  background-color: #364150;
+}
+.header {
+  display: flex;
+  justify-content: center;
+  margin-top: 3%;
+}
+.contentlogin {
+  background-color: #eceef1;
+  -webkit-border-radius: 7px;
```

Comment:
![image](https://user-images.githubusercontent.com/28482442/45595562-b19f6580-b9b6-11e8-975f-a5c92f83025c.png)

Border radius is already part of most browsers, you don't need these lines.